models/marts/educ_serv/staging/stg_type_mesure.sql

```diff
@@ -15,6 +15,34 @@ GNU Affero General Public License for more details.
 You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #}
-select fiche, id_eco, string_agg(type_mesure, ', ') as type_mesure
-from {{ ref("i_gpm_e_mesures") }}
-group by fiche, id_eco
+with
+    indice as (
+        select distinct
+            type_mesure,
+            case
+                when
+                    type_mesure = '11'
```

Comment:
@sadqim  suggestion, tu peux faire un type_mesure IN ('11', ...)
C<est plus rapide car ca passe par un hash vs faire des disjonctions

Reply:
done!